erp/schema.py

```diff
@@ -558,6 +569,19 @@
         "nullable_bool": True,
         "warn_if": False,
     },
+    "entree_dispositif_appel_type": {
+        "is_a11y": True,
+        "label": "Type de dispositif d'appel à l'entrée",
+        "help_text": mark_safe(
+            "Quel(s) type(s) de dispositifs sont présents&nbsp;?"
```

Comment:
Don't hesitate to re-precise *Quels types de dispositifs **d'appel** sont présents*